### PR TITLE
Fix radio button group tokens based on figma file and update tokens build script

### PR DESCRIPTION
**Pull request by gregaubert**
https://www.figma.com/file/3bo9DCW9JD52Lc21Fcyea8/(Echoes)-Components?type=design&node-id=15-2267&mode=design&t=y4SaQ7XkLllhUgno-4

### CODE REVIEW

**Comment by gregaubert** [comment]
@marciopmoreira6 I renamed the base token group to have it all lower case like the other token groups.
Also can you use this branch instead of the one you pushed to check that the changes are fine in figma? I'll drop the old branch you re-created when we were on zoom.

File: design-tokens/build.js [hunk 1]
```diff
@@ -30,14 +30,16 @@ const BASE_TOKENS_NAME = `${NAME_PREFIX}base`;
 const CUSTOM_TRANSFORM_GROUP = 'sonar-design-tokens';
 const CUSTOM_FILTER_NO_COLOR = 'sonar-no-color';
 const CUSTOM_FILTER_THEMED_TOKENS = 'sonar-themed-tokens';
-const themeDefinitions = JSON.parse(fs.readFileSync(`${DESIGN_TOKENS_PATH}/$themes.json`, 'utf-8'));
-const licenceHeader = fs.readFileSync(`config/license/LICENSE-HEADER.txt`, 'utf-8');
+const LICENCE_HEADER = fs.readFileSync(`config/license/LICENSE-HEADER.txt`, 'utf-8');
+
+const TOKEN_GROUPS = JSON.parse(fs.readFileSync(`${DESIGN_TOKENS_PATH}/$themes.json`, 'utf-8'));
+const THEMED_TOKEN_GROUPS = TOKEN_GROUPS.filter(({ group }) => group === 'Themes');
```
Comment (jeremy-davis-sonarsource) [inline design-tokens/build.js:36]: NP: I prefer to use Capitalized constants for true hard-coded values. Here we're reading external data, so I think it makes sense to have "normal" constants. Corollary: don't use the `licenseHeader` directly, but pass it as an argument

**Review by jeremy-davis-sonarsource — COMMENTED** [review]
Just one code-style nitpick, WDYT?

File: design-tokens/build.js (hunk 1 above)
Comment (gregaubert) [reply design-tokens/build.js:36]: Ok it make sense

**Review by jeremy-davis-sonarsource — APPROVED** [review]
Don't forget to squash!

**Pull request merged**